container.go

```diff
@@ -28,12 +28,22 @@ import (
 	"github.com/containerd/containerd/cio"
 	"github.com/containerd/containerd/containers"
 	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/containerd/images"
 	"github.com/containerd/containerd/oci"
+	"github.com/containerd/containerd/runtime/v2/runc/options"
 	"github.com/containerd/typeurl"
 	prototypes "github.com/gogo/protobuf/types"
+	ver "github.com/opencontainers/image-spec/specs-go"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/pkg/errors"
 )
 
+const (
+	checkpointImageNameLabel       = "image.name"
```

Comment:
Is there prior use of this label anywhere? This seems different from `org.opencontainers.image.ref.name` in that it only applies to an index annotation rather than on a manifest descriptor. Is there any reason not to also namespace this?

Reply:
I didn't know we added this, I thought it was `image.name` in OCI for a while but we can change it to the fully qualified name is that is the new one